extensions/vitest.config.js

```diff
@@ -23,9 +24,9 @@
  * @see https://vitest.dev/config/
  */
 const config = {
+  root: PACKAGE_ROOT,
   test: {
     globals: true,
-    environment: 'jsdom',
```

Comment:
hello, why is it removed ?

Reply:
the environment is node, not jsdom, I can make a dedicated PR if needed